src/commands/branch.js

```diff
@@ -0,0 +1,32 @@
+const { flags } = require('@oclif/command');
+const AbstractAuthenticatedCommand = require('../abstract-authenticated-command');
+
+class BranchCommand extends AbstractAuthenticatedCommand {
+  async runIfAuthenticated() {
+    // eslint-disable-next-line no-unused-vars
+    const parsed = this.parse(BranchCommand);
+  }
+}
+
+BranchCommand.description = 'Create a new branch or list your existing branches';
```

Comment:
Well, I added a comment yesterday on this, but since github was down-ish, I guess it was not saved.

All the other commands does not end with a final ".", so I removed those (For consistency). But maybe I was wrong?

Reply:
Nor heroku nor NPM have dots. So let's keep it that way